index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <main id="main">
+      <section id="title">
+        <h2>Dr. Norman Borlaug</h2>
+        <p>The man who saved a billion lives</p>
+      </section>
+
+      <div id="img-div">
+        <img id="image" src="./assets/images/tribute-page-main-image.jpeg" />
+        <small id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </small>
+      </div>
```

Comment:
Great Job you did here, however, please use semantic tags to replace the div tag, since this is an image with caption, you can use ` <figure></figure>` tag to wrap both the image and the captions and replace the `<small></small>`  tag with ` <figcaption> </figcaption> `, this way, your code will follow the semantics rules and also great for accessibilities.